src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-list/rbd-list.component.html

```diff
@@ -29,6 +29,26 @@
        [innerHtml]="'Only available for RBD images with <strong>fast-diff</strong> enabled'"></div>
 </ng-template>
 
+<ng-template #provisionedNotAvailableTooltipTpl
+             let-row="row">
+  <span *ngIf="row.disk_usage === null; else provisioned"
```

Comment:
```suggestion
  <span *ngIf="!row.disk_usage; else provisioned"
```

Reply:
@nizamial09 what if `disk_usage == 0`?

Reply:
Hmm you are right. then `null` should be the one! Thanks @epuertat